src/codegen/arm64/macro-assembler-arm64.h

```diff
@@ -926,7 +925,8 @@ class V8_EXPORT_PRIVATE MacroAssembler : public MacroAssemblerBase {
   // to 64-bit before calling this function.
   void Switch(Register scratch, Register value, int case_value_base,
               Label** labels, int num_labels);
-#if V8_TARGET_CHERI
+
+  // NOTE(zyj20): These are for CHERI architectures
```

Comment:
Since we have shims for these, you could probably remove the note and instead just do something like:

```
// Start of CHERI methods
...
// End of CHERI methods
```

Do you think that would make sense?